src/tailwind/defaultConfig.ts

```diff
@@ -56,7 +60,8 @@ export default {
       "high-deal": "#E34A15",
       "fair-deal": "#EAA403",
       "good-deal": "#81B040",
-      "great-deal": "#149246",
+      "great-deal": colors.greenDark,
+      "green-dark": colors.greenDark,
```

Comment:
thinking about it... I think I'd prefer to stick to using the `great-deal` class instead of inventing our own names for colors that should be properly defined in the styleguide. I think it's wrong for the designer to use this color for this purpose in the first place

up to you.. :)

Reply:
I checked it back with Enrique... (also synced quickly on the colours...)
![image](https://user-images.githubusercontent.com/3371760/120606424-789dcc80-c44f-11eb-9795-9f509f4320e6.png)